com.github.AberothCommunity.Aberoth.metainfo.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.AberothCommunity.Aberoth</id>
+  <name>Aberoth</name>
+  <developer_name>Aberoth</developer_name>
+  <summary>Free 8-bit MMORPG</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://aberoth.com/help.php</project_license>
+  <launchable type="desktop-id">com.github.AberothCommunity.Aberoth.desktop</launchable>
+  <url type="homepage">https://aberoth.com/help.php</url>
+  <url type="help">https://aberoth.com/help.php</url>
+  <update_contact>40026133+ashnel3@users.noreply.github.com</update_contact>
+  <description>
+    <p>Aberoth is a fantasy massively multi-player online role playing game.</p>
+    <p>Unofficial flatpak</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://aberoth.com/media/castle.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://aberoth.com/media/stony_field.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://aberoth.com/media/orc_prison_skull.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://aberoth.com/media/grand_shaman.png</image>
+    </screenshot>
+  </screenshots>
+  <categories>
+    <category>Game</category>
+  </categories>
```

Comment:
Add release

Reply:
I'm not sure how to do this, the game only has dated releases.

Reply:
> I'm not sure how to do this, the game only has dated releases.

https://freedesktop.org/software/appstream/docs/sect-Metadata-Releases.html